wp-content/themes/illdy-child/layout/js/translations_page.js

```diff
@@ -82,18 +82,32 @@ var TranslationsPage = (function(window, $) {
       // Give sidebar the list of languges
       this.sidebar.setList(this.translationsData);
 
-      // Pre-select a language if there's something in the query string
-      // TODO: Look for the 'lang' parameter on the query string specifically
+      // Default to English resources. Will trigger sidebar.render();
+      this.sidebar.setSelected('en');

```

Comment:
I would reccommend having a variable for selected language that defaults to en and then update the value in 91-98 and then just run `this.sidebar.setSelected()` at the very end because in this case sidebar.render() will trigger twice according to the comments.